docs: PE-8512 Add Kairos v4.0.3 UKI known issue (#10147)
spectrocloud/librarium: docs/docs-content/release-notes/known-issues.md

> | <!--https://spectrocloud.atlassian.net/browse/PE-8512 --> Kairos upgrade for Edge workflows from v3.5.9 to v4.0.3 does not apply to [Unified Kernel Images (UKI)-based Trusted Boot images](../clusters/edge/trusted-boot/trusted-boot.md), which remain on Kairos v3.5.9. This version difference does not impact functionality.                                                                                                                                                                                                                                                                                                                                                                                                                | No workaround available.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | May 3, 2026        | Edge                                      |

Reviewer: ```suggestion
| <!--https://spectrocloud.atlassian.net/browse/PE-8512 --> [Unified Kernel Images (UKI)-based Trusted Boot images](../clusters/edge/trusted-boot/trusted-boot.md) are excluded from the Kairos v3.5.9 to v4.0.3 upgrade for Edge workflows, and remain on Kairos v3.5.9. This version difference does not impact functionality.                                                                                                                                                                                                                                                                                                                                                                                                                | No workaround available.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | May 3, 2026        | Edge                                      |
```

Reviewer: I found the sentence a little hard to follow, so I made this suggestion.
